target/linux/ath79/tiny/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /lib/functions/migrations.sh
+
+board=$(board_name)
+boardonly="${board##*,}"
+
+case "$board" in
+tplink,tl-wr802n-v2)
+	migrate_leds "^$boardonly:=tp-link:"
```

Comment:
This is correct at the moment, but the whole file would become obsolete if the DTS trigger works.

Reply:
If we stick to not defining trigger altogether (see above), this still has merit to migrate any configuration users might have made previously.

Reply:
Fair point. Didn't think of that.
The only question is whether this also applies for a ar71xx->ath79 migration, where the user is encouraged to upgrade without keeping settings anyway. So, in general I'd follow your argument, but for this specific case I think the user should be "made aware" that something changed (i.e. the "recommended" LED behavior), and given the chance to start from the "new default" setup.

However, this is just for your consideration, if you prefer to keep the led_migration I will be fine with that as well.

Reply:
I think it doesn't hurt to have this, and personally wouldn't like this kind of encouragement. I guess that a lot of people will just forget about setting this up anyway :)